static/openapi/prerendering-spec.json

```diff
@@ -4335,525 +4691,631 @@
         }
       }
     },
-    "/api/prerender/v1/origins/{origin}/prefetch/urls": {
+    "/api/prerender/v1/history/render": {
       "get": {
-        "summary": "Get prefetch URLs",
-        "tags": ["Prefetch URLs"],
-        "description": "Returns the latest prefetch URLs associated with the origin.",
+        "summary": "Get render history data",
+        "tags": ["Prerender History"],
+        "description": "Returns historical data for web metrics, with time-range filtering and optional response code criteria. This endpoint requires the from and to query parameters, indicating the time range as epoch seconds. Optional filters include responseCode for HTTP status, and pagination controls through limit and offset. Data can be optionally scoped to a specific origin if the origin path parameter is included. The response comprises an array of historical records, detailing bytes sent, response time, device type, and response codes. Authentication is handled via the customerId header.",
         "parameters": [
           {
-            "schema": { "type": "number", "default": 0 },
+            "schema": { "type": "number" },
             "in": "query",
-            "name": "offset",
+            "name": "from",
+            "required": true,
+            "description": "Epoch time in seconds."
+          },
+          {
+            "schema": { "type": "number" },
+            "in": "query",
+            "name": "to",
+            "required": true,
+            "description": "Epoch time in seconds."
+          },
+          {
+            "schema": { "type": ["number", "string"] },
+            "in": "query",
+            "name": "responseCode",
             "required": false,
-            "description": "Pagination offset."
+            "description": "Response code for filtering. It can be specific or in range, e.g., 200-310."
```

Comment:
This is strange.  For one API above, you can use 3xx, but here you use range.  We will want to be consistent and do it one way.